modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java

```diff
@@ -1223,6 +1227,10 @@ public BlogsEntry updateEntry(
 			serviceContext.getAssetLinkEntryIds(),
 			serviceContext.getAssetPriority());
 
+		// Comment
+
+		_subscribeDiscussion(entry);
```

Comment:
Hi @epgarcia,

Do we want to subscribe on update too? I think this might cause an issue in this scenario:

1. User creates the entry. 
2. User decides they don't want to receive notifications about it and unsubscribes (using the My Subscriptions option in the user menu).
3. User updates the entry.
4. User is angry because they start getting notifications again even though they unsubscribed.

What do you think?

Reply:
Hi @4lejandrito,

Yes, it makes sense. Should I remove line 1132 and send the PR again? Or can you do it as part of your review? 

Thanks

Reply:
Cool, it's OK, I'll do it.